os/drivers/wireless/scsc/misc/mx_mmap.c

```diff
@@ -557,11 +557,6 @@ int mx_gdb_close(struct file *filp)
 
 	sem_post(&mx_dev->data_wait);
 	return OK;
-
-	filp->f_priv = NULL;
-	mx_dev->filp = NULL;
```

Comment:
I think these two lines shoud be moved above line 559 i.e., above `return OK;` statement